tmol/io/pose_stack_from_rosettafold2.py

```diff
@@ -130,6 +130,80 @@ def canonical_form_from_rosettafold2(seq, xyz, chainlens):
     )
 
 
+def pose_stack_to_rosettafold2(seq, xyz, chainlens, pose_stack):
```

Comment:
This name no longer really fits the file. How should we rename the function/file?